src/randoop/operation/NonreceiverTerm.java

```diff
@@ -164,17 +164,22 @@ public Object getValue() {
   }
 
   /**
-   * Returns constraint to represent new reference to this statement
+   * {@inheritDoc}
+   * @return type of value.
    */
+  @Override
   public Class<?> getOutputType() {
     return this.type;
   }
 
   /**
-   * Returns the appropriate PrimitiveOrStringOrNullDeclInfo representative of 
+   * createCanonicalTerm returns a NonreceiverTerm representative of 
    * the specified class c.
+   * 
+   * @param c type of value desired.
+   * @return a {@link NonreceiverTerm} with a canonical representative of the given type.
    */
-  public static NonreceiverTerm nullOrZeroDecl(Class<?> c) {
+  public static NonreceiverTerm createCanonicalTerm(Class<?> c) {
```

Comment:
I find the name "canonical" confusing here; it makes me think of canonicalization, which is not what this does.  For this one, I prefer the original name.  In the change, the documentation and name doesn't indicate that it will always be null or zero.